spec/model/car_cleaner_spec.rb

```diff
@@ -0,0 +1,13 @@
+require_relative '../../model/car_cleaner'
+
+describe CarCleaner do
+  let(:car_cleaner_empty) { CarCleaner.new }
+  let(:car_cleaner_with_data) { CarCleaner.new(["Test", "Another Test"]) }
+  
+  it "stores an array as a queue" do
+
+    expect(car_cleaner_empty.queue).to eq []
+    expect(car_cleaner_with_data.queue). to eq ["Test", "Another Test"]
+  end
+
```

Comment:
I'll write you here how I would have done these tests:

```
context 'with empty data' do
  subject { CarCleaner.new }

  it 'stores an array as a queue' do
    response = subject

    expect(response.queue).to eq []
  end
end


context 'with data' do
  subject { CarCleaner.new(['Test', 'Another Test']) }

  it 'stores an array as a queue' do
    response = subject

    expect(response.queue). to eq ['Test', 'Another Test']
  end
end
```

Since you are testing 2 different behaviours it is ok to have 2 different contexts instead of testing 2 behaviours in the same it. Let me know if it's not clear enough.

Reply:
It's clear and it makes more sense. Thank you!